examples/accessing_tiles.rs

```diff
@@ -20,63 +20,95 @@ fn startup(
     let texture_handle = asset_server.load("tiles.png");
     let material_handle = materials.add(ColorMaterial::texture(texture_handle));
 
-    let mut map = Map::new(MapSettings::new(
-        UVec2::new(4, 4),
-        UVec2::new(32, 32),
-        Vec2::new(16.0, 16.0),
-        Vec2::new(96.0, 256.0),
-        0,
-    ));
-    let map_entity = commands.spawn().id();
-    map.build(
+    // We can create maps by using the LayerBuilder
+    // LayerBuilder creates the tile entities and makes sure they are attached to chunks correctly.
+    // It also provides a way of accessing and viewing tiles during the creation phase.
+    // Once a LayerBuilder is passed to the map_query.create_layer function it is consumed and
+    // can no longer be accessed.
+    // Layer builder accepts a generic bundle that must implement the `TileBundleTrait`.
+    // This is used internally to access the tile in the bundle to attach the chunk correctly.
+
+    // Layer Entity
+    let layer_entity = commands.spawn().id();
+
+    // Create the layer builder
+    let mut layer_builder = LayerBuilder::<TileBundle>::new(
         &mut commands,
-        &mut meshes,
-        material_handle,
-        map_entity,
+        layer_entity,
+        LayerSettings::new(
```

Comment:
I think that `LayerBuilder::new()` should take `layer_id` as a parameter, and we also should probably move `layer_id` out of `LayerSettings` since it's not really data that can/should be reusable between layers.